dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -197,10 +235,38 @@ var dallinger = (function () {
     return deferred;
   };
 
+  /**
+   * Convenience method for making an AJAX ``GET`` requst to a specified route.
+   * Includes error handling to help ensure workers are compensated
```

Comment:
I think this error handling/compensation stuff is now dependent on calling `dallinger.error()` with the `AjaxRejection` object you get back from the failed `Deferred`(?) See my comments below in the Deferred section.

Reply:
We had to back off from some of the automagical error handling because it moved too much control away from the caller.